src/main/java/org/icgc/argo/program_service/converter/FromProtoProgramConverter.java

```diff
@@ -0,0 +1,60 @@
+package org.icgc.argo.program_service.converter;
```

Comment:
Why don't we combine the FromTo and ToProto into one `Mapper`? http://mapstruct.org/documentation/stable/reference/html/#basic-mappings

Reply:
Originally it was used to make it easier to read, but since everything is type checked...not really that valuable. will change back.